src/injectors-cache.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * Contains functions that inject dynamic content.
+ */
+
+import * as brotli from './brotli-wasm-wrapper';
+import {
+  ContentEncoding,
+  HeaderKeys,
+  IncomingCloudflarePropertiesExtended,
+  supportsBrotli,
+} from './headers';
+
+const INJECTORS_CACHE_NAME = 'injectors';
+const textEncoder = new TextEncoder();
+
+let injectorsCache_: Cache;
+/** */
+async function openInjectorsCache(): Promise<Cache> {
+  if (!injectorsCache_) {
+    injectorsCache_ = await caches.open(INJECTORS_CACHE_NAME);
```

Comment:
Does this need to be memoized?

Reply:
Not sure! I'm guessing each request creates an entirely separate environment, so this wouldn't get memoized between requests. It would shave off reopening the cache when we need to read and then save to it in the same request